developer_docs/how_to_create_a_plugin.asciidoc

```diff
@@ -2477,6 +2477,21 @@ gemspec file list. The .po and .mo files are important in development
 and production environments respectively, so must both be shipped in the
 gem.
 
+[[registering-translations]]
+=== Registering Translations
```

Comment:
Below we have a line that says "requires Foreman 1.12+ so set requires_foreman" and I think that's a good practice.